rubygem-smart_proxy_dhcp_remote_isc/rubygem-smart_proxy_dhcp_remote_isc.spec

```diff
@@ -0,0 +1,84 @@
+# Generated from smart_proxy_dhcp_remote_isc-0.0.1.gem by gem2rpm -*- rpm-spec -*-
+%global gem_name smart_proxy_dhcp_remote_isc
+%global plugin_name dhcp_remote_isc
+
+%global foreman_proxy_dir %{_datarootdir}/foreman-proxy
+%global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
+%global foreman_proxy_settingsd_dir %{_sysconfdir}/foreman-proxy/settings.d
+
+Name: rubygem-%{gem_name}
+Version: 0.0.2
+Release: 1%{?foremandist}%{?dist}
+Summary: Smart-Proxy dhcp module provider for NFS-accessible ISC dhcpd installations
+Group: Applications/Internet
+License: GPLv3
+URL: https://github.com/theforeman/smart_proxy_dhcp_remote_isc
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+Requires: foreman-proxy >= 1.15.0
+Requires: ruby(release)
+Requires: ruby
+Requires: ruby(rubygems)
+BuildRequires: rubygems-devel
```

Comment:
Missing `BuildRequires: ruby(release)`, may have been removed when removing the EL6 conditional.